cleanrl_utils/same_model_vector_env.py

```diff
@@ -0,0 +1,34 @@
+from copy import deepcopy
+
+import gymnasium as gym
+import numpy as np
+
+
+# Only for gymnasium v1.0.0
+class SameModelSyncVectorEnv(gym.vector.SyncVectorEnv):
```

Comment:
This should probably be called `SameStepModeSyncVectorEnv` or we just shift to gymnasium v1.1.0

Reply:
https://farama.org/Vector-Autoreset-Mode